apps/dashboard/test/helpers/batch_connect/sessions_helper_test.rb

```diff
@@ -53,28 +53,10 @@ class BatchConnect::SessionsHelperTest < ActionView::TestCase
     assert_equal delete_session_title, button['title']
   end
 
-  test 'relaunch should ignore sessions when session is not completed' do
-    OodCore::Job::Status.states.each do |state|
-      next if state == :completed
-
-      status_array = []
-      relaunch(status_array, create_session(state))
-      assert_equal [], status_array
-    end
-  end
-
-  test 'relaunch should ignore sessions when session application is not valid' do
-    status_array = []
-    relaunch(status_array, create_session(:completed, valid: false))
-    assert_equal [], status_array
-  end
```

Comment:
These tests were testing that interface specifically which I changed. I got rid of the behavior that modifies that `status_array` in favor of a more straight forward templating scheme.